generate_styles.py

```diff
@@ -4,8 +4,12 @@
 import glob
 import sys
 
-colors = {
+'''colors = {
```

Comment:
I would just delete this rather than commenting/stringing it out - if we want to see what it was before, that's what Git is for